### ISSUE TITLE
-ss option and --log does not work correctly sometimes.

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#95]
Reproduced Environment:
  Docker on macOS 10.12.06 Sierra + Docker(18.06.0-ce-mac70)
  container: ubuntu:18.04
  tmux version: 2.4
  bash version: 3.2.0

```
$ xpanes --debug -ss --log=~/logs --log-format="[:ARG:]" -I@ -d -c "echo TEST | sed s/TEST/@/" AAAA BBBB CCCC DDDD

# Expected
## ~/logs/AAAA-1 => AAAA
## ~/logs/BBBB-1 => BBBB
## ~/logs/CCCC-1 => CCCC
## ~/logs/DDDD-1 => DDDD

# Actual (sometimes)
## ~/logs/AAAA-1 => AAAA
## ~/logs/BBBB-1 => BBBB
## ~/logs/CCCC-1 => !!EMPTY!!
## ~/logs/DDDD-1 => DDDD

# $ cat ~/logs/CCCC-1
# => nothing
```

Same problem seems happened on the TravisCI [link](https://travis-ci.org/greymd/tmux-xpanes/builds/410534727).

`tmux pipe-pane` is used for the logging feature of tmux-xpanes.
During the development for version 3.1.0, I noticed, the logging feature does not work correctly sometimes.
It seems that, running multiple pipe-pane processes does not work correctly.

Here is PoC.
1. Creating multiple named pipes.
2. Split tmux panes and start the processes to wait each named pipe updated.
3. Run `tmux pipe-pane` to redirect each pane's stdout.
4. Notify all the named pipes.
5. Some results of the `pipe-pane` is empty. Frequently.

```bash
#!/bin/bash

# Try this script on the docker container.
# Tested on macoOS High Sierra + Ubuntu 18.04 docker container.

rm -f "$HOME"/work/*

tmux -S "$HOME"/work/sess new-session -d

printf "%s\\n" A B C D | while read -r f;do
  mkfifo "$HOME/work/$f"
  echo "start to wait: mkfifo $HOME/work/$f"
  _pane_id=$( tmux -S "$HOME"/work/sess splitw -P "grep -q 1 $HOME/work/$f"$'\n'"echo 'This is $f'" )
  echo "start logging: pipe-pane cat >> $HOME/work/$f.log"
  tmux -S "$HOME"/work/sess pipe-pane -t "$_pane_id" "cat >> $HOME/work/$f.log"
done

printf "%s\\n" A B C D | while read -r f;do
  echo "notify to $f"
  printf "%s\\n" 1 > "$HOME/work/$f" &
done

sleep 5

printf "%s\\n" "$HOME/work"/*.log | while read -r file;do
  echo "$file"
  cat "$file"
done

# $ bash fifo.sh
# start to wait: mkfifo /home/docker/work/A
# start logging: pipe-pane cat >> /home/docker/work/A.log
# start to wait: mkfifo /home/docker/work/B
# start logging: pipe-pane cat >> /home/docker/work/B.log
# start to wait: mkfifo /home/docker/work/C
# start logging: pipe-pane cat >> /home/docker/work/C.log
# start to wait: mkfifo /home/docker/work/D
# start logging: pipe-pane cat >> /home/docker/work/D.log
# notify to A
# notify to B
# notify to C
# notify to D
# /home/docker/work/A.log
# This is A
# /home/docker/work/B.log
# This is B
# /home/docker/work/C.log <<<<<<<<<<<<<<<<<<< EMPTY !!!
# /home/docker/work/D.log <<<<<<<<<<<<<<<<<<< EMPTY !!!
```

**Comment by greymd** [comment]
Even `sleep` command is inserted any part, the problem is reproduced..

**Comment by greymd** [comment]
It seems that, this is caused by https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=865756

**Comment by greymd** [comment]
Same issue does not reproduced in tmux 2.7.
Therefore, it would be known bug in tmux.
(`--log` feature does not work in tmux 2.3  due to [Known Bug](https://github.com/greymd/tmux-xpanes/wiki/Known-Bugs). That's why status is green.)

**Comment by greymd** [comment]
This issue seems fixed in tmux 2.6 and more.
https://travis-ci.org/greymd/tmux-xpanes/builds/410575182

**Comment by greymd** [comment]
Let me update Known Bug page.
https://github.com/greymd/tmux-xpanes/wiki/Known-Bugs

**Comment by greymd** [comment]
Fixed at [v3.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v3.1.0)
